src/initial_read_module.f90

```diff
@@ -1568,6 +1569,8 @@ subroutine read_input(start, start_L, titles, vary_mu,&
     CGreset               = fdf_boolean('AtomMove.ResetCG',          .false.)
     MDn_steps             = fdf_integer('AtomMove.NumSteps',     100        )
     MDfreq                = fdf_integer('AtomMove.OutputFreq',    50        )
+    XSFfreq               = fdf_integer('AtomMove.XsfFreq',    MDfreq        )
+    XYZfreq               = fdf_integer('AtomMove.XyzFreq',    MDfreq        )
```

Comment:
Some minority considerations, 
How about make `AtomMove.XsfFreq` and` AtomMove.XyzFreq` to `AtomMove.OutputFreqXSF` and` AtomMove.OutputFreqXYZ`? As sub option of `AtomMove.OutputFreq` and also consistent to `AtomMove.WriteXSF` and `AtomMove.WriteExtXYZ`.
Is it okay to keep default value to '1', since it is usually considered to print out 'XYZ' or 'XSF' each step in common DFT calculations?

Reply:
>How about make AtomMove.XsfFreq and AtomMove.XyzFreq to AtomMove.OutputFreqXSF and AtomMove.OutputFreqXYZ? As sub option of AtomMove.OutputFreq and also consistent to AtomMove.WriteXSF and AtomMove.WriteExtXYZ.

Strictly speaking, they are not a sub option of `AtomMove.OutputFreq`, since they could be written independently from the other files (`md.frames`, etc.). 

>Is it okay to keep default value to '1', since it is usually considered to print out 'XYZ' or 'XSF' each step in common DFT calculations?

I am not sure what is the best option. (see also https://github.com/OrderN/CONQUEST-release/issues/255#issuecomment-1704625057).  For MD in general, it is not common to write every step. For geometry relaxation (with cg, sqnm, etc.), writing every step is probably useful.